alembic/versions/2020_06_29_11_37_00-544564b4e738.py

```diff
@@ -0,0 +1,24 @@
+"""Add locked, lock_reason, locked_by and locked_by_id to mod
```

Comment:
```suggestion
"""Add indexed Blog.announcement
```

Reply:
Haha, you can tell how _this_ file was created!